provider/cluster/service.go

```diff
@@ -259,6 +259,7 @@ func (s *service) run(deployments []ctypes.Deployment) {
 		key := deployment.LeaseID()
 		mgroup := deployment.ManifestGroup()
 		s.managers[key] = newDeploymentManager(s, deployment.LeaseID(), &mgroup)
+		_ = s.bus.Publish(event.LeaseAddFundsMonitor{LeaseID: deployment.LeaseID()})
```

Comment:
Ignoring this error seems like a bad idea? Just log it if something happens, so we can track down what is failing.